nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/util/FTPTransfer.java

```diff
@@ -606,7 +606,7 @@ private FTPClient getClient(final FlowFile flowFile) throws IOException {
             client.setAutodetectUTF8(useUtf8Encoding);
         }
 
-        client.connect(inetAddress, ctx.getProperty(PORT).evaluateAttributeExpressions(flowFile).asInteger());
+        client.connect(remoteHostName, ctx.getProperty(PORT).evaluateAttributeExpressions(flowFile).asInteger());
```

Comment:
This change as-is needs more work.  We create the inetAddress in previous lines using this remoteHostName value.  This change just ignores all those lines.  So we need to understand why those other lines aren't needed anymore and remove them.  There is a proxyHostname property available too.  Makes me wonder if we're using the wrong value here.  Genearlly we need to tighten this up. 

In any event it appears this is your first time attempting to contribute code.  Much appreciated either way

Reply:
@joewitt thanks for answer.

Can you remember, why are you did resolving `inetAddress`? 
As i see in blame it's your code.
![image](https://user-images.githubusercontent.com/5636129/120192155-81aa5600-c223-11eb-9f28-a255a03b153d.png)

Reply:
@joewitt  ?

Reply:
I'm not sure.  But if we dont need that code at this point it can/should be removed.

Reply:
check this plz

i did it

Reply:
I dont understand the removal of the utf8 handling either.  And the multiple JIRA references here make it unclear to me what this PR Is precisely attempting to do.